README.md

```diff
@@ -150,6 +150,7 @@ fn main() {
 When using C++ library compilation switch, the `CXX` and `CXXFLAGS` env
 variables are used instead of `CC` and `CFLAGS` and the C++ standard library is
 linked to the crate target.
+Remember that C++ does name mangling so `extern "C {...}` might be required to enable rust linker to find your functions.
```

Comment:
I think there's a missing `"` here, and perhaps the `{...}` can be dropped? There's a few different styles of doing this in C++ I believe but `extern "C"` should be the common denominator.

Reply:
I updated the sentence ! Thanks.